### DISCUSSION TITLE
Hangi kapıyı hangi kilit açar

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#209] [Zeka Sorusu]
Bir koridorda sarı, kırmızı, yeşil ve mavi renklerde 4 adet kilit ile bu kilitleri açan 4 adet anahtar vardır (Her anahtar farklı bir kilide aittir) . Hangi anahtarın hangi kilidi açtığı bilinmiyor. Buna göre her bir anahtarın hangi kilidi açacağını en az kaç denemede kesin olarak belirleyebiliriz?

**Comment by Alperg123456** [comment]
1. kapıda 3 deneme yaparız üçü de uymazsa 4.yü denememize gerek yok zaten                                                                  3 deneme yaptık
2. kapıda 2 deneme yeterli çünkü birini 1'e taktık eğer denediğimiz 2 anahtar da olmazsa 3.yü demeye gerek yok           2 deneme yaptık
3. kapıda 1 deneme yeterli çünkü 2 anahtar zaten taktık biri tutmazsa diğeri zaten olur                                                       1 deneme yaptık
4. kapıda ise denemeye gerek yok çünkü zaten 1 anahtar kaldı                                                                                               0 deneme yaptık

Yani toplam 6 denemede çözdük
